src/main/java/com/minecolonies/coremod/configuration/ConfigurationHandler.java

```diff
@@ -143,7 +136,33 @@ private static synchronized void loadConfiguration()
             }
         }
     }
-    
+
+    /**
+     * load configuration related to Path finding.
+     */
+    private static synchronized void loadPathFindingConfigurations()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "loadPathFindingConfigurations" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)